modules/nw-configuring-qinq-sriov-proc.adoc

```diff
@@ -0,0 +1,243 @@
+<2> Specifies the VLAN protocol to assign to the VF. Allowed values are `802.1ad` and `802.1q`. The default is `802.1q`. 
+
+. To create the object, enter the following command:
++
+[source,terminal]
+----
+$ oc create -f <name>.yaml
+----
+
+. Optional: To confirm that the `NetworkAttachmentDefinition` object that is associated with the `SriovNetwork` object that you created in the previous step exists, enter the following command. Replace `<namespace>` with the networkNamespace you specified in the `SriovNetwork` object.
++
+[source,terminal]
+----
+$ oc get net-attach-def -n default
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME                   AGE
+sriovnetwork-1ad-810   29s
+----
+
+. Create a Network Attachment Definition with an inner VLAN. The inner VLAN is often referred to as the C-tag, or `Customer Tag`, as it belongs to the Network Function.
+
+.. Using the following YAML example, create a file named `nad-cvlan100.yaml`:
++
+[source,yaml]
+----
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  name: nad-cvlan100
+  namespace: default
+spec:
+  config: |
+    {
+      "cniVersion": "0.3.1",
+      "name": "vlan-100",
+      "plugins": [
+        {
+          "type": "vlan",
+          "master": "net1", 
+          "vlanId": 100,
+          "linkInContainer": true, 
+          "ipam": {"type": "static"}
+        }
+      ]
+    }
+----
+
+.. Apply the YAML file by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f nad-cvlan100.yaml
+----
+
+. Optional: To confirm that the `NetworkAttachmentDefinition` object that is associated with the `SriovNetwork` object that you created in the previous step exists, enter the following command. Replace `<namespace>` with the networkNamespace you specified in the `SriovNetwork` object.
++
+[source,terminal]
+----
+$ oc get net-attach-def -n default
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME              AGE
+sriov-intel       58m
+nad-cvlan100      54m
+----
+
+.Verification
+
+Verify QinQad is active on the node by following this procedure: 
+
+. Obtain a list of cluster nodes: 
++
+[source,terminal]
+----
+$ oc get nodes
+----
+
+. Enter into a debug session on the target node, the node with the NIC attached. This step instantiates a debug pod called <node_name>-debug:
++
+[source,terminal]
+----
+$ oc debug node/my-cluster-node
+----
+
+. Set `/host` as the root directory within the debug shell. The debug pod mounts the host’s root file system in /host within the pod. By changing the root directory to /host, you can run binaries contained in the host’s executable paths:
++
+[source,terminal]
+----
+# chroot /host
+----
+
+. To display information about network interfaces, including their status and configuration, run the following command on the physical interface the virtual functions are created on:
++
+[source,terminal]
+----
+# ip -d link show ens5f0
+----
++
+.Example output
++
```

Comment:
if you don't create  pod, you won't see any changes. These are only apply when the pod is created.